.travis.yml

```diff
@@ -9,13 +9,13 @@ env:       TARGET=x86_64-unknown-linux-gnu
 matrix:
   include:
     - env: TARGET=i686-unknown-linux-gnu
-    - env: TARGET=x86_64-unknown-linux-gnu
+    - env: TARGET=x86_64-unknown-linux-gnu RUST_BETA=1
```

Comment:
nit: the TRAVIS_RUST_VERSION variable will contain the string 'beta' or 'nightly' when 'rust: beta' or 'rust: nightly' is used here. You can use that var instead of defining a new one.